auth/src/main/java/org/aerogear/mobile/auth/ConnectionBuilderForTesting.java

```diff
@@ -1,112 +0,0 @@
-package org.aerogear.mobile.auth;
-
-import android.annotation.SuppressLint;
-import android.net.Uri;
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
-import android.util.Log;
-
-import net.openid.appauth.Preconditions;
-import net.openid.appauth.connectivity.ConnectionBuilder;
-
-import java.io.IOException;
-import java.net.HttpURLConnection;
-import java.net.URL;
-import java.security.KeyManagementException;
-import java.security.NoSuchAlgorithmException;
-import java.security.cert.X509Certificate;
-import java.util.concurrent.TimeUnit;
-
-import javax.net.ssl.HostnameVerifier;
-import javax.net.ssl.HttpsURLConnection;
-import javax.net.ssl.SSLContext;
-import javax.net.ssl.SSLSession;
-import javax.net.ssl.TrustManager;
-import javax.net.ssl.X509TrustManager;
-/**
- * An example implementation of {@link ConnectionBuilder} that permits connecting to http
- * links, and ignores certificates for https connections. *THIS SHOULD NOT BE USED IN PRODUCTION
- * CODE*. It is intended to facilitate easier testing of AppAuth against development servers
- * only.
- */
-public class ConnectionBuilderForTesting implements ConnectionBuilder {
```

Comment:
I was wondering about removing that thing, too 

why was it added ? should it live in tests ?  @ziccardi ?

Reply:
It was there because, without that, untrusted certificates are... untrusted ;)
I don't think we need it in the tests. It was just meant to make life easier in development environments.